# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/mac/moz.build
# Commit: 0441d1553b6b
# Full Hash: 0441d1553b6bbf99d711300680eccd335e0b3545
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2020-07-03 03:56:55
# Regressor Bug: 1649575
# File Overlap Count: 1
# Description:
#   Bug 1649575 - Part 3: Introduce MOXTextMarkerDelegate. r=morgan
#   
#   The delegate is associated with accessible documents (either local or remote).
#   
#   I made a separate protocol for all the text marker stuff as it really is a seperate API.
# ==============================================================================

diff -r 65f8e705d730 -r 0441d1553b6b accessible/mac/moz.build
--- a/accessible/mac/moz.build	Thu Jul 02 21:24:39 2020 +0000
+++ b/accessible/mac/moz.build	Thu Jul 02 21:33:07 2020 +0000
@@ -19,6 +19,7 @@
     'MacUtils.mm',
     'MOXAccessibleBase.mm',
     'MOXMathAccessibles.mm',
+    'MOXTextMarkerDelegate.mm',
     'MOXWebAreaAccessible.mm',
     'mozAccessible.mm',
     'mozActionElements.mm',